docs/extensibility/how-to-roundtrip-VSIXs.md

```diff
@@ -0,0 +1,204 @@
+
+
+## Update Extension Manifest
+
+#### 1. Installation Targets
+We need to tell Visual Studio what versions to target for building a VSIX.  Typically, these references are either to version 14.0 (Visual Studio 2015) or version 15.0 (Visual Studio 2017).  In our case, we want to build a VSIX that will install an extension for both, so we need to target both versions.  If you want your VSIX to build and install on versions earlier than 14.0, this can be done by setting the earlier version number; however, version 10.0 and earlier are no longer supported.
+
+* Open the source.extension.vsixmanifest file in Visual Studio.
+* Open the Install Targets tab
+* Change the Version Range to [14.0, 16.0).  The " [ " tells Visual Studio to include 14.0 and all versions past it.  The " ) " tells Visual Studio to include all versions of 15.0 up to, but not including version 16.0.
+* Save all changes and close all instances of Visual Studio
+
+![Installation Targets Image](media/visual-studio-installation-targets-example.png)
+
+#### 2.	Adding Prerequisites to the extension.vsixmanifest file 
+
+Prerequisites are a new feature with Visual Studio 2017.  In this case we need the Visual Studio Core Editor as a Prerequisite. Since the Visual Studio 2015 VSIX designer does not handle the new Prerequisites section, you’ll need to edit this part manually in the XML code.  Alternatively, you can open Visual Studio 2017 and use the updated manifest designer to insert the Prerequisites.  To do this manually:
+
+* Navigate to the project directory in File Explorer
+* Open the extension.vsixmanifest file with a text editor 
+* Add the following tag:
+
+	```<Prerequisites>```  
+	```<Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />```  
+	```</Prerequisites>```  
+* Save and close the file.
+
+>**Note:** If you choose to accomplish this with the VSIX designer in Visual Studio 2017, you will need to manually edit the Prerequisite version to ensure it is compatible with all versions of Visual Studio 2017.  This is because the designer will insert the minimum version as your current version of Visual Studio (e.g. 15.0.26208.0).  However, since other users may have an earlier version, you will want to manually edit this to 15.0.
+
+At this point, your manifest file should look something like this:
+
+![Prerequisites Example](media/visual-studio-prerequisites-example.png)
+
+## Modify the project file (myproject.csproj)
+
+It is highly recommended to have a reference to a modified csproj open while doing this step.  You can find several examples [here](https://github.com/Microsoft/VSSDK-Extensibility-Samples).  Select any extensibility sample, find the csproj file for reference and execute the following steps:
+
+* Navigate to the project directory in File Explorer
+* Open the myproject.csproj file with a text editor (Notepad ++ is recommended)
```

Comment:
No - we do not recommend Notepad++. If anything, we could recommend Visual Studio Code. But notepad is just fine.
Best we give no recommendation at all.